lstchain/scripts/lstchain_r0g_to_r0v.py

```diff
@@ -116,6 +126,15 @@ def main():
             n_tiles = header["NAXIS2"]
             rows_per_tile = header["ZTILELEN"]
 
+            if args.fix_pixel_status:
+                r0v_input = True
+                if "LSTDVR" in header:
+                    r0v_input = header["LSTDVR"]
+                else:
+                    r0v_input = False
```

Comment:
```suggestion
                r0v_input = header.get("LSTDVR", False)
```

Reply:
If it is not R0V the file may not even contain the LSTDVR field , this is why I did it like that. I think header.get it would yield an error which would be less clear than the current one